src/reactioncoordinate.cpp

```diff
@@ -328,19 +328,21 @@ MoleculeProperty::MoleculeProperty(const json &j, Space &spc) : ReactionCoordina
     else if (property == "Rinner") {
         dir = j.at("dir");
         indexes = j.value("indexes", decltype(indexes)());
-        assert(indexes.size() == 2 && "An array of 2 indexes should be specified.");
-        function = [&spc, &dir = dir, i = indexes[0], j = indexes[1]]() {
+        assert(indexes.size() >= 3 && "An array of at least 3 indexes should be specified.");
```

Comment:
better to use `throw ConfigurationError("...")` as `assert` is only run when compiler in Debug mode.